src/development/LocalTest/Services/Storage/Implementation/InstanceEventRepository.cs

```diff
@@ -3,6 +3,7 @@
 using LocalTest.Configuration;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
```

Comment:
Please try to use `System.Text.Json` instead.
`System.Net.Http.Json` also have a few nice classes/extensions that can be usefull here.

Reply:
Newtonsoft is already used other places in the same source file. The problem is not releated to the json converter. The problem is that the old code referred to a model class that is at another level than incoming json string. The incoming string matches the C# class InstanceEventList. The old code used List<InstanceEvent>, which is a property of InstanceEventList.